src/test/app/Flow_test.cpp

```diff
@@ -1254,6 +1254,34 @@ struct Flow_test : public beast::unit_test::suite
             ter(temBAD_PATH));
     }
 
+    template<class... T>
+    void testWithFeats(bool haveFlow, T&&... fs)
```

Comment:
We can get rid of the `bool` parameter and check for `haveBool` directly in the param pack, but it requires the changes in this PR: https://github.com/ripple/rippled/pull/2216

I'd like to make that change, but I think the easiest is to wait for both PRs to be merged and to created another PR later with the change.